analysed-packages/hdparm/version-9.63/hdparm-9.63-SPDX2TV.spdx

```diff
@@ -0,0 +1,1418 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_hdparm-9.63.tar.gz_1725012976.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Sascha Dierberg
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of hdparm 9.63.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-08-30T10:16:16Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: hdparm-9.63
+PackageFileName: hdparm-9.63.tar.gz
+SPDXID: SPDXRef-upload606
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: 1e3381c01ad3c9ae1a18dd5b739fdaedcf19bfdf
+PackageChecksum: SHA1: c8e24233b4d29506907ad2bf0725f793106c3a78
+PackageChecksum: SHA256: 70785deaebba5877a89c123568b41dee990da55fc51420f13f609a1072899691
+PackageChecksum: MD5: cea97ea2aa164f66817adc98c6814280
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-BSD-style
```

Comment:
I think the package license declared is: LicenseRef-BSD-style-b2c8dff3432f7dbc09d6fef9564cad6a and not LicenseRef-BSD-style